examples/neptune_tracking/README.md

```diff
@@ -0,0 +1,145 @@
+# 🏋️ Track your runs with Neptune
+
+[Neptune](https://neptune.ai/product/experiment-tracking) is a
+popular platform that you would normally use in the iterative ML experimentation
+phase to track and visualize experiment results. Neptune can also
+track and visualize the results produced by your
+automated pipeline runs, as you make the transition towards a more production
+oriented workflow.
+
+## 🗺 Overview
+This example showcases how to use the Neptune Experiment Tracker flavor to log and visualize information from your pipeline steps (e.g. models, parameters, metrics).
+
+We'll be using the [MNIST](http://yann.lecun.com/exdb/mnist/) dataset and
+will train a classifier using [Tensorflow (Keras)](https://www.tensorflow.org/).
+We will run two experiments with different parameters (epochs and learning rate)
+and log these experiments into a neptune platform. 
+
+In the example script, the [neptune-tensorflow-keras](https://docs.neptune.ai/integrations/keras/) is
+used within the training step to directly hook into the TensorFlow training.  
+It will log out all relevant parameters, metrics and output files Automatically. . Additionally, we explicitly log the test accuracy within the evaluation step.
+
+Note that despite `neptune` being used in different steps within a pipeline, ZenML handles initializing `neptune` and ensures the experiment name is the same as the pipeline name, and the experiment run is the same name as the pipeline run name. This establishes a lineage between pipelines in ZenML and experiments in `neptune`.
+
+# 🖥 Run it locally
+
+### 📄 Prerequisites 
+
+### Set up a free Neptune account
+To get this example running, you need to set up a Neptune account. You can do this for free [here](https://app.neptune.ai/register).
+
+### Pre-requisites
+In order to run this example, you need to install and initialize ZenML:
+
+```shell
+# install CLI
+pip install "zenml[server]"
+
+# install ZenML integrations
+zenml integration install tensorflow sklearn neptune
```

Comment:
(Maybe we can remove sklearn here as well? If we do need it, it also needs to be included in the `setup.sh`